investigacion/Investigación-de-usuarios.md

```diff
@@ -0,0 +1,25 @@
+# Investigación de usuarios #
+
+El objetivo de esta encuesta ha sido obtener información sobre el comportamiento de los usuarios de redes sociales respecto a contenido político.
```

Comment:
Aquí tendría sentido que hicierais alguna mención a cómo llegáis a plantearos esta encuesta: a través de la elaboración del canvas, los supuestos etc., y cómo esta es la manera de confirmar (o desmentir) algunos de los supuestos en los que habéis basado vuestro planteamiento de sitio web.